README.md

```diff
@@ -34,6 +34,7 @@
       <ul>
         <li><a href="#linux">Linux</a></li>
         <li><a href="#windows">Windows</a></li>
+        <li><a href="#termux">Termux</a></li>
```

Comment:
Change **Termux** to **Android (Termux)**. Because we are providing installation steps for different operating systems. It will make more sense to someone who doesn't already know about **Termux** app.

Reply:
Done